amazon-machine-learning.md

```diff
@@ -0,0 +1,18 @@
+## Name of Talk 
```

Comment:
Missing title?

Reply:
Doh! Well, that just shows why PR review is always a good idea. Just added.